src/benchmark/micro_benchmark_basic_fixture.cpp

```diff
@@ -38,4 +40,10 @@ void MicroBenchmarkBasicFixture::_clear_cache() {
   micro_benchmark_clear_cache();
 }
 
+uint32_t MicroBenchmarkBasicFixture::_align_to_pagesize(const uint32_t buffer_size_mb, const uint32_t page_size) {
```

Comment:
can you document, what align to page size does?
Either in the pr or in the code